snippets/RPNSolver.md

```diff
@@ -0,0 +1,35 @@
+### RPNSolver
+
+Solves the given reverse polish notation
+```js
+const RPNSolver = RPN => {
+  const isNumeric = str => !isNaN(parseFloat(str)) && isFinite(str);
+  const isOperator = str => ['*','-','+','/','**'].includes(str)
+  let stack = [];
+  let solve  = RPN.replace(/\^/g,'**').split(/\s+/g);
+  solve.forEach(symbol => {
+    if(isNumeric(symbol)) {stack.push(symbol)}
+          else if (isOperator(symbol)){
+          a = stack.pop();
+          b = stack.pop();
+          if(symbol === "+") {
```

Comment:
This `if... else if...` chain can be written as a switch or even using some `Array.prototype` method (maybe a bit too complex in that case).